src/main/java/Parser.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    Scanner sc = new Scanner(System.in);
+
+    public static boolean handleCommand(String command, ListOfTasks xs) {
+        while (!command.equalsIgnoreCase("bye")) {
+            try {
+                    if (command.equalsIgnoreCase("list")) {
+                        xs.listOut();
+                    } else if (command.contains("done")) {
+                        try {
+                            if (command.equals("done ") || command.equals("done")) {
```

Comment:
Are you sure you can't make it more efficient than setting up two conditions just for one space?